### ISSUE TITLE
Thoughts on using a component lib?

### ISSUE THREAD

**Issue by jschr** [KristerV/heliumpay-budgetweb-frontend#5]
Would still be great to get some nice branding and mockups eventually but until we can fund that work I think the frontend could benefit from using a component library for faster dev of upcoming features. The current user branch suffers from verbose forms and lack of nice UX for form fields and buttons.

I've looked through a few, this looks like a solid one since it has components for comments and menus: https://react.semantic-ui.com

Alternatives: https://github.com/brillout/awesome-react-components#ui-frameworks

**Comment by KristerV** [comment]
Yeah Semantic was my top pic actually. I don't think I got it to work with next.js in a clean way. Also since I didn't want JS dependence that time I decided it's not worth the hassle. Also if a designer comes in it's going to be harder to implement a custom design.

But then again forms are a hassle to do manually so I really don't mind if we end up using Semantic.
